panther_lights/CMakeLists.txt

```diff
@@ -16,12 +16,20 @@ include_directories(include)
 
 add_executable(driver_node src/driver_node_main.cpp src/driver_node.cpp
                            src/apa102.cpp)
-
+target_include_directories(
+  driver_node
+  PUBLIC ${image_transport_INCLUDE_DIRS} ${panther_gpiod_INCLUDE_DIRS}
+         ${panther_msgs_INCLUDE_DIRS} ${rclcpp_INCLUDE_DIRS}
+         ${sensor_msgs_INCLUDE_DIRS})
 ament_target_dependencies(driver_node image_transport panther_gpiod
                           panther_msgs rclcpp sensor_msgs)
 
 add_executable(dummy_scheduler_node src/dummy_scheduler_node_main.cpp
                                     src/dummy_scheduler_node.cpp)
+target_include_directories(
+  dummy_scheduler_node
+  PUBLIC ${image_transport_INCLUDE_DIRS} ${rclcpp_INCLUDE_DIRS}
+         ${sensor_msgs_INCLUDE_DIRS})
```

Comment:
Again, why do we modify the lights driver library linkage in the simulation task?

Reply:
Similar thing like above. It not build all the time so I make few changes thats, work form me. Thanks for this.